cupy/testing/_helper.py

```diff
@@ -1,37 +1,41 @@
+from __future__ import annotations
+
 import contextlib
 import inspect
-from typing import Callable
 import unittest
-from unittest import mock
 import warnings
+from collections.abc import Callable
+from importlib import metadata
```

Comment:
```suggestion
from importlib.metadata import PackageNotFoundError, version
```

Reply:
I'd like to use `metadata.version` since the word `version` is too generic and prone to name conflict.

Reply:
That seems reasonable

Was mainly scratching my head thinking, "What is `metadata`? Have we defined it?" This despite knowing we were using `importlib.metadata` and having used it myself periodically

So was debating whether we go the verbose route and use `importlib.metadata` in a few places or `import` names

That said, no strong feelings here. Happy to go with whatever consensus is on this point

Reply:
I see, then how about using the fully-qualified name: `importlib.metadata.version` ?

Reply:
SGTM

Reply:
Done.